owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -110,6 +110,11 @@ class ReleaseNotesViewModel(
                 subtitle = R.string.release_notes_4_3_0_subtitle_display_name_truncation,
                 type = ReleaseNoteType.BUGFIX
             ),
+            ReleaseNote(
+                title = R.string.release_notes_4_3_0_title_show_app_provider_icon_from_endpoint,
+                subtitle = R.string.release_notes_4_3_0_subtitle_show_app_provider_icon_from_endpoint,
+                type = ReleaseNoteType.BUGFIX
```

Comment:
```suggestion
                type = ReleaseNoteType.ENHANCEMENT
```

Reply:
This issue is a `Feature Request` so the type of release note must be `ENHANCEMENT`

Reply:
Yeah, rigth